docs/layouts/shortcodes/generated/prometheus_push_gateway_reporter_configuration.html

```diff
@@ -12,7 +12,7 @@
             <td><h5>deleteOnShutdown</h5></td>
             <td style="word-wrap: break-word;">true</td>
             <td>Boolean</td>
-            <td>Specifies whether to delete metrics from the PushGateway on shutdown.</td>
+            <td>Specifies whether to delete metrics from the PushGateway on shutdown. Flink will try its best to delete the metrics but this is not guaranteed. See here <a href="https://issues.apache.org/jira/browse/FLINK-13787">for more details.</a></td>
```

Comment:
The html file is not in sync with PrometheusPushGatewayReporterOptions.java

Reply:
thx, I'm too careless